src/ray/common/ray_config_def.h

```diff
@@ -479,7 +479,8 @@ RAY_CONFIG(int64_t, ray_syncer_polling_buffer, 5)
 RAY_CONFIG(uint64_t, gcs_service_address_check_interval_milliseconds, 1000)
 
 /// The batch size for metrics export.
-RAY_CONFIG(int64_t, metrics_report_batch_size, 100)
+/// Normally each metrics is about < 1KB. 1000 means it is around 1MB.
```

Comment:
what does the original batch size field control if there's already such a config?

Reply:
that one is not used anywhere (maybe except Ant). And based on my initial research, 100 is anyway too small.